docs/userGuide/syntax/mathformulae.mbdf

```diff
@@ -0,0 +1,99 @@
+## Math Formulae
+Markbind supports typesetting TeX math equations. [KaTeX](https://katex.org) is used as a fast math renderer.
+
+<box type="info">
```

Comment:
let's present this vital piece of information about the feature in plain markup, without the box which readers might just skip over